spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -113,16 +113,21 @@
 
   describe '#show' do
     let!(:transcription) { create(:transcription) }
+    let(:user) { create(:user, :admin) }
 
-    describe 'roles' do
-      before do
-        allow(controller).to receive(:current_user).and_return user
-        get :show, params: { id: transcription.id }
-      end
+    before do
+      allow(controller).to receive(:current_user).and_return user
+      get :show, params: { id: transcription.id }
+    end
 
+    it 'serializes the updated_at date in the "Last-Modified" header' do
+      expect(response.header['Last-Modified']).to eq(transcription.updated_at.httpdate)
```

Comment:
is `httpdate` the expected string format the `updated_at` value is serialized as? via https://github.com/zooniverse/tove/blob/cb5e163f6be7dd2634fef9822a89a7e770964e26/app/controllers/transcriptions_controller.rb#L24

https://ruby-doc.org/stdlib-2.7.0/libdoc/time/rdoc/Time.html#method-c-httpdate
`Returns a string which represents the time as RFC 1123 date of HTTP-date defined by RFC 2616:`

Reply:
Yes, it's serialized as `httpdate`. When a date value is passed to the `response.last_modified` setter, it's transformed into the `httpdate` format. See here:
https://github.com/rails/rails/blob/v5.2.3/actionpack/lib/action_dispatch/http/cache.rb#L56

Which means I need to test against `transcription.updated_at.httpdate` in the spec.

Reply:
nice!